repo_guidelines.md

```diff
@@ -49,13 +55,39 @@ See our [proposed linting standard](./protocol/linting.md) for further details.
+We love improvements to our tools! EDGI has general [guidelines for contributing][edgi-contributing] and a [code of conduct][edgi-conduct] for all of our organizational repos.
+
+## Here are some notes specific to this project:
+Use this section for any contributing guidelines specific to the repo. For example:
+* [Web monitoring's processing repo guidelines](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/CONTRIBUTING.md) mentions which issues should be made on that repo versus on the main web monitoring repo
+* In the [Web Monitoring umbrella repo's contribution guidelines](https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md), a style guide, testing best practices, and notes on how work is distributed (process) are laid out
+* The guidelines on [the EDGI Scripts repo](https://github.com/edgi-govdata-archiving/edgi-scripts/blob/master/CONTRIBUTING.md), [video call landing page](https://github.com/edgi-govdata-archiving/video-call-landing-page/blob/master/CONTRIBUTING.md) and [100 Days](https://github.com/edgi-govdata-archiving/100days/blob/master/CONTRIBUTING.md) discuss important notes on how the continuous integration is configured
+* [The website's contributing guidelines](https://github.com/edgi-govdata-archiving/edgi-website/blob/master/CONTRIBUTING.md) tell you where to find the appropriate Slack channel
+
+<!-- Links -->
+   [edgi-conduct]: https://github.com/edgi-govdata-archiving/overview/blob/master/CONDUCT.md
+   [edgi-contributing]: https://github.com/edgi-govdata-archiving/overview/blob/master/CONTRIBUTING.md
 ```
 
-### License & Copyright README block
+### Readme Template
+
+Each Readme must include:
+* [License & Copyright block](#license--copyright-readme-block) templated below
+* Code of Conduct [![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](https://github.com/edgi-govdata-archiving/overview/blob/master/CONDUCT.md) badge
+* A basic description of what the repo is or does
+* A "How to start contributing to this repo" section including developer setup (if relevant), Slack channel, contact person, etc.
+
+Suggestions for additional components of Readmes:
+* A "How to use" section if the repo's project is a tool or website
+* Highlight "ready" label on issues to mean "this is an issue that is ready to work on and needs an owner"
+* Additional badges at the top including [![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](https://www.firsttimersonly.com/) linking to "good first issue" label, code quality indicators
+* "[All contributors](https://github.com/kentcdodds/all-contributors#emoji-key)" listing
```

Comment:
Is it worth pointing out that we have some non-standard practices around all-contributors?

- Compact representation without avatars (less visual noise; easier to focus on contributions),
- Icons are links with title attributes for accessibility,
- Alphabetical order by name (surname if we have it) (to eliminate implied ranking), and
- Presence in the list is optional and up to the contributor (not everyone wants to be listed — we offer, but do not add unless someone explicitly says yes).

Reply:
Yes, that would be useful. Is this the right place to put that (is there a better place)? I think this might be the best place we have, but it feels like a lot to put in a bullet point

Reply:
Yeah, I don’t if there’s anywhere better. There are always sub-bullets. ¯\\\_(ツ)\_/¯

Reply:
Also all the points above about our customizations are things that I pretty much have dictated/just done. I think they are all good things and have a reason, but also we’ve never really *discussed* that if others think differently.

Reply:
Acknowledged, but honestly I think dictating the customization with those rules is fine/if we don't like them, this PR is a good place to bring it up & have the discussion.